src/WebJobs.Script/Eventing/EventSources.cs

```diff
@@ -14,5 +14,6 @@ public static class EventSources
         public const string ScriptFiles = "ScriptFiles";
         public const string Rpc = "Rpc";
         public const string Worker = "Worker";
+        public const string WorkerProcess = "WorkerProcess";
```

Comment:
What's the difference between the Worker source and WorkerProcess source?

Reply:
Added WorkerProcess  source to specifically capture process shutdown/exit events.  Worker source is for rest of the language worker events. Here is an example https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script/Rpc/FunctionRpcService.cs#L60